MachObfuscator/Options/Options.swift

```diff
@@ -1,16 +1,29 @@
 import Foundation
 
+class EraseSectionConfiguration {
```

Comment:
Don't you think it would be more natural to use a struct here? You could when provide the initialiser in extension.
```suggestion
struct EraseSectionConfiguration {
     var sectionName: String
     var segmentName: String
}

extension EraseSectionConfiguration
      init(sectionDef: String) {
         let sectionParts = sectionDef.split(separator: ",", maxSplits: 2)
         self.init(sectionName: String(sectionParts[1]), segmentName: String(sectionParts[0]))
     }
 }
```

Reply:
Your version is more Swift-y :) Changed.